### PR TITLE
Max

### PR DESCRIPTION
**Pull request by mzhang2021**
@ahsu1230 
@tonywu315
@miniviolin
@mythicat554

### CODE REVIEW

File: orion/README.md [hunk 1]
```diff
@@ -91,6 +91,9 @@ mysql.server start
 mysql.server stop
 mysql.server restart
 ```
+Alternatively, if that doesn't work, you can use the MySQL Notifier app that comes with installation and appears in the taskbar:
+
+![alt text](https://github.com/ahsu1230/mathnavigatorSite/blob/max/orion/Untitled.png)
```
Comment (ahsu1230) [inline orion/README.md]: Could we give this image a more useful name?
Maybe "onboarding_mysql_notifier.png"?

**Comment by ahsu1230** [comment]
It's also good practice to give your commit message a more meaningful title :)

Just keep it short and somewhat descriptive like, "Add MySql Notifier to Orion Onboarding"

**Comment by mzhang2021** [comment]
I modified the change.

**Comment by ahsu1230** [comment]
I just created an onboarding folder in the orion directory. Do you mind moving this image into there? Once you do, this change looks good to me :+1:

**Pull request merged**